core/src/main/scripts/importer/cbioportalImporter.py

```diff
@@ -419,8 +419,7 @@ def usage():
                            '--command [%s] --study_directory <path to directory> '
                            '--meta_filename <path to metafile>'
                            '--data_filename <path to datafile>'
-                           '--study_ids <cancer study ids for remove-study command, comma separated>'
-                           '--properties-filename <path to properties file> ' % (COMMANDS)), file=OUTPUT_FILE)
+                           '--study_ids <cancer study ids for remove-study command, comma separated>' % (COMMANDS)), file=OUTPUT_FILE)
```

Comment:
I don't quite understand this. Are we stopped supporting loading the customized property file?
Why do we remove this `'--properties-filename <path to properties file> '`?

Reply:
>It looks to be not used.
That was also my conclusion.